front-end/public/index.html

```diff
@@ -12,7 +12,7 @@
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/front-end-manifest/
```

Comment:
not biggy - this got changed as well lol..

Reply:
Woops this was me 🤣  did a replace all and this got included